auth/src/main/java/com/firebase/ui/auth/data/model/Resource.java

```diff
@@ -18,10 +18,29 @@
     private final T mValue;
     private final Exception mException;
 
+    /**
+     * Creates a successful Resource&lt;Void&gt;.
```

Comment:
`Creates a successful, empty Resource.` would be clearer to read in the source code.